src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div key={movie.imdbId} className="card" data-cy="Movie">
+        <MovieCard movie={movie} />
+      </div>
+    ))}
+  </div>
```

Comment:
Use semantic tags `ul` + `li`
`li` shoud be a part of MovieCard 
```suggestion
  <ul className="movies">
    {movies.map(movie => (
        <MovieCard movie={movie}  key={movie.imdbId} />
    ))}
  </ul>
```